### PR TITLE
Sync fetch-configlet.ps1 script

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR updates the bin/fetch-configlet.ps1 script to the latest version. This fixes the issue mentioned in this forum post: https://forum.exercism.org/t/problem-running-the-powershell-version-of-fetch-configlet/8383.

References https://github.com/exercism/configlet/issues/839

### CODE REVIEW

File: exercises/practice/reverse-string/.approaches/span/content.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # Span&lt;T&gt;
```
Comment (wolf99) [inline exercises/practice/reverse-string/.approaches/span/content.md:1]: Seems like some other change to an unrelated file is added in this PR. Expected or incorrect?

**Review by wolf99 — CHANGES_REQUESTED** [review]

File: exercises/practice/reverse-string/.approaches/span/content.md (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/practice/reverse-string/.approaches/span/content.md:1]: Whoa. Missed this comment. It was unintended but correct
